src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -3,108 +3,15 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import inspect
-import functools
 import os
 
-from .base import ScenarioTest, execute
+from azure_devtools.scenario_tests.base import ScenarioTest
```

Comment:
Conceptually, you are not importing the same ScenarioTest. I think this should still be `from .base import ScenarioTest` to import the one from CLI, and not the base one (unless you are 100% sure the base class is enough here).